requirements.txt

```diff
@@ -2,19 +2,25 @@ colander==1.0
 cornice==1.1.0
 dealer==2.0.4
 iso8601==0.1.10
+newrelic==2.56.0.42
 PasteDeploy==1.5.2
+psycopg2==2.6.1
 pyramid==1.5.7
 pyramid-multiauth==0.5.0
 python-dateutil==2.4.2
+raven==5.7.2
 redis==2.10.3
 repoze.lru==0.6
-requests==2.7.0
+requests==2.8.0
 simplejson==3.8.0
 six==1.9.0
-structlog==15.2.0
+statsd==3.1
```

Comment:
Actually this file is meant to be a reference in case the software doesn't work anymore when you try to install it.
It will not be used to actually install the cliquet version.